mysql-9.1.yaml

```
@@ -132,6 +132,59 @@ subpackages:
       runtime:
         - "${{package.name}}-oci-entrypoint"
 
+  - name: ${{package.name}}-bitnami-compat
+    dependencies:
+      provides:
+        - mysql-bitnami-compat=${{package.full-version}}
+      runtime:
+        - bash
+        - busybox
+        - coreutils
+        - ini-file
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: mysql
+          # NOTE: 9.0 is the last version published by bitnami
+          version-path: 9.0/debian-12
+          commit: 2a237f1b8024064a5d1d9e92c7f3e06a79e3f44d
+      - name: Create symlinks
+        runs: |
+          mkdir -p "${{targets.contextdir}}/opt/bitnami/mysql/bin"
+          for file in mysql $(find "${{targets.destdir}}/usr/bin" -type f -exec basename {} \;) ; do
+            ln -sf /usr/bin/$file "${{targets.contextdir}}/opt/bitnami/mysql/bin/$file"
+          done
+
+          mkdir -p "${{targets.contextdir}}/opt/bitnami/mysql/lib"
+          ln -s /usr/lib/${{package.name}}/plugin "${{targets.contextdir}}/opt/bitnami/mysql/lib/plugin"
+          ln -s /usr/lib/private "${{targets.contextdir}}/opt/bitnami/mysql/lib/private"
+          ln -s /usr/share/${{package.name}} "${{targets.contextdir}}/opt/bitnami/mysql/share"
+
+          mkdir -p "${{targets.contextdir}}/usr/share/${{package.name}}"
+          ln -s /usr/share/${{package.name}} "${{targets.contextdir}}/usr/share/${{package.name}}/${{package.name}}"
+
+          mkdir -p "${{targets.contextdir}}/opt/bitnami/mysql/conf/bitnami"
+          install -m644 bitnami-my.cnf "${{targets.contextdir}}/opt/bitnami/mysql/conf/my.cnf"
+          cp -pfR "${{targets.contextdir}}/opt/bitnami/mysql/conf" "${{targets.contextdir}}/opt/bitnami/mysql/conf.default"
+
+          mkdir -p "${{targets.contextdir}}/opt/bitnami/mysql/logs"
+          ln -sf /proc/1/fd/1 "${{targets.contextdir}}/opt/bitnami/mysql/logs/mysqld.log"
```

nit: for better readability.
```suggestion
          ln -sf /dev/stdout "${{targets.contextdir}}/opt/bitnami/mysql/logs/mysqld.log"
```

---

I added the exact same symlink that upstream is doing. I'd rather keep this as is unless you're strong about this change?